ecs-agent/daemonimages/csidriver/driver/node_test.go

```diff
@@ -0,0 +1,161 @@
+//go:build linux && unit
+// +build linux,unit
+
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package driver
+
+import (
+	"context"
+	"errors"
+	"os"
+	"testing"
+
+	"github.com/container-storage-interface/spec/lib/go/csi"
+	"github.com/golang/mock/gomock"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+var (
+	volumeID = "voltest"
+	nvmeName = "/dev/disk/by-id/nvme-Amazon_Elastic_Block_Store_voltest"
+)
+
+func TestNodeGetVolumeStats(t *testing.T) {
+	testCases := []struct {
+		name     string
+		testFunc func(t *testing.T)
+	}{
+		{
+			name: "success normal",
+			testFunc: func(t *testing.T) {
+				mockCtl := gomock.NewController(t)
+				defer mockCtl.Finish()
+
+				mockMounter := NewMockMounter(mockCtl)
+				VolumePath := "./test"
+				err := os.MkdirAll(VolumePath, 0644)
+				if err != nil {
+					t.Fatalf("fail to create dir: %v", err)
+				}
+				defer os.RemoveAll(VolumePath)
+
+				mockMounter.EXPECT().PathExists(VolumePath).Return(true, nil)
```

Comment:
Why do we need this mock? We are creating a real file so real implementation should work fine?

Reply:
It's required for `mockMounter`

Reply:
We need this file to skip this check `IsBlockDevice`

Reply:
For consistency of ebs csi driver, we will use the mockMounter.